index.html

```diff
@@ -1608,7 +1608,35 @@ <h3>Defining 'string'</h3>
 	</div>
 </section>
 
+<section id="char_whitespace" class="subtopic">
+	<h3>Whitespace characters</h3>
+	
+	<div class="xref"><span class="seealso">See also</span>
+	<p>[[[#markup_identifiers]]].</p>
+	</div>
 
+	<p>
+		Whitespace characters are characters that represent horizontal or vertical space in typography. Whitespace characters can have different visual effects: some whitespace characters have no visible effect, while others represent larger, smaller, or variable amounts of space on the page.
+	</p>
+	
+		<div class="req" id="char_define_whitespace">
+			<p>
+				Some specifications that define formal languages (examples include <a href="https://html.spec.whatwg.org/multipage/syntax.html#syntax">HTML</a> or <a href="https://www.w3.org/TR/css-syntax/#whitespace">CSS</a>) will choose to specify ASCII whitespace as part of their grammar. Specifications that deal more generally with text will choose to follow Unicode's definition instead.
+			</p>
+			<p class="advisement">Specifications that mention whitespace characters SHOULD explicitly state which characters are whitespace characters, by referring to the Unicode <a href="https://www.unicode.org/reports/tr44/#White_Space">White_Space</a> property or <code>General_Category=Z</code> [[UAX44]], or specifying the specific code points.</p>
```

Comment:
This still seems a bit "chewy". Perhaps:

> Specifications that use the term "whitespace" SHOULD explicitly define what the term means in their context. For most specifications, this should refer to the Unicode `White_Space` property or to `General_Category=Z`. Otherwise the specific code points should be listed.

We still don't say when to use the property or when to use the character class.